test/SortPerformanceEstimatorTest.cpp

```diff
@@ -9,12 +9,16 @@
 
 #include "engine/SortPerformanceEstimator.h"
 #include "util/Log.h"
+#include "util/MemorySize/MemorySize.h"
 #include "util/Random.h"
 
+using namespace ad_utility::memory_literals;
+
 TEST(SortPerformanceEstimator, TestManyEstimates) {
   // only allow the test to use 1 Gig of RAM
   auto allocator = ad_utility::AllocatorWithLimit<Id>{
-      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1ull << 30ul)};
+      ad_utility::makeAllocationMemoryLeftThreadsafeObject(
+          ad_utility::MemorySize::bytes(1ull << 30ul))};
```

Comment:
Should be `1_GB`

Reply:
Done, see [c992264](https://github.com/ad-freiburg/qlever/pull/1112/commits/c992264350fe820fccb3480e44cb7a04cb5e1e27).